fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -1215,23 +1221,34 @@ private SqlQueryData processTokenParm(Class<?> resourceType, QueryParameter quer
 
             // Build this piece of the segment:
             // (P1.PARAMETER_NAME_ID = x AND
-            this.populateNameIdSubSegment(whereClauseSegment, queryParm.getCode(), tableAlias);
+            this.populateNameIdSubSegment(whereClauseSegment, queryParmCode, tableAlias);
 
             whereClauseSegment.append(AND).append(LEFT_PAREN);
             for (QueryParameterValue value : queryParm.getValues()) {
+                appendEscape = false;
+
                 // If multiple values are present, we need to OR them together.
                 if (parmValueProcessed) {
                     whereClauseSegment.append(OR);
                 }
 
                 whereClauseSegment.append(LEFT_PAREN);
-                
+
                 if (Modifier.IN.equals(queryParm.getModifier()) || Modifier.NOT_IN.equals(queryParm.getModifier())) {
                     populateValueSetCodesSubSegment(whereClauseSegment, value.getValueCode(), tableAlias);
                 } else {
                     // Include code
                     whereClauseSegment.append(tableAlias + DOT).append(TOKEN_VALUE).append(operator).append(BIND_VAR);
-                    bindVariables.add(SqlParameterEncoder.encode(value.getValueCode()));
+                    if (LIKE.equals(operator)) {
```

Comment:
Under what scenario would we have a LIKE here?

I was actually thinking line 1195 should always return EQ, so maybe we could just remove that variable altogether?  Maybe in the case of `:in` and `:not-in` (and soon `:above`/`:below`) it could be something different, but I think we handle those separately today.

Reply:
never mind, now I get it.  we set LIKE because we want it to behave similar to our normal (no-modifier) string search behavior.